conf/routes

```diff
@@ -56,7 +56,7 @@ GET         /photo/encounter/:id                               @femr.ui.controll
 GET         /triage/:id                                        @femr.ui.controllers.TriageController.indexPopulatedGet(id: Integer)
 GET         /triage                                            @femr.ui.controllers.TriageController.indexGet()
 POST        /triage                                            @femr.ui.controllers.TriageController.indexPost(id: Integer)
-POST        /triage/:id                                        @femr.ui.controllers.TriageController.deletePatientPost(id: Integer)
+POST        /triage/:id                                        @femr.ui.controllers.TriageController.deletePatientPost(id: Integer, deleteByUserID: Integer)
```

Comment:
Adding a route is one way to do this, but the deletePatientPost() method has a currentUser variable which will always contain the ID of the user (just use currentUser.getId())